.github/workflows/rust.yml

```diff
@@ -180,3 +180,24 @@ jobs:
       - name: Deploy develop # unused
         if: github.ref == 'refs/heads/develop'
         run: bash scripts/deploy_docker.sh develop
+
+  autodeploy:
```

Comment:
We would be missing a deployment for `tags` which are the ones that represent our `production` deployments.

Reply:
There is no auto-deploy implemented for production, because of that there is no place for tagged deployments. Maybe this could change in the coming future...

Reply:
@jpalvarezl this update is just to trigger a hook (authenticated) to notify that a new `staging` image is available (that's how I understood it)

Reply:
Exactly, although the conditions could be more than just a new image, other projects use to have a `develop` branch for `dev-` deployments